### PR TITLE
chore: add hubspot company and contact

### PR DESCRIPTION
**Pull request by tomkit**
- companies and contacts for hubspot

## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

File: apps/api/mappers/hubspot.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import type { HubspotCompany, HubspotContact } from '@/types/hubspot';
+import type { SupaglueRecord } from '@supaglue/core/lib';
+
+export const toHubspotCompany = (record: SupaglueRecord): HubspotCompany => {
```
Comment (asdfryan) [inline apps/api/mappers/hubspot.ts:4]: How are you doing associations?

**Review by asdfryan — COMMENTED** [review]
Did you test this locally?

**Comment by tomkit** [comment]
> Did you test this locally?

Yea hitting the endpoints worked

File: apps/api/mappers/hubspot.ts (hunk 1 above)
Comment (tomkit) [reply apps/api/mappers/hubspot.ts:4]: Returning the member-attribute association on `contact` and leaving the raw `associations` in the `raw_data`

**Review by asdfryan — APPROVED** [review]

**Pull request merged**